common/src/main/java/com/wynntils/screens/WynntilsMenuScreen.java

```diff
@@ -207,6 +226,27 @@ private void renderButtons(PoseStack poseStack, int mouseX, int mouseY) {
 
             if (hovered) {
                 this.hovered = button;
+            }
+
+            if (!button.dynamicTexture()) {
```

Comment:
What is this check for?

Reply:
Some textures have 2 states, and they "animate" when hovered. Some are stateless.